src/Adaptor.js

```diff
@@ -166,6 +166,17 @@ function handleOptions(options) {
   return (options && options.setNull) || "'undefined'";
 }
 
+function escapeQuote(stringExp) {
+  if (typeof stringExp === 'object' && stringExp !== null) {
```

Comment:
Would love to chat about this for a moment—looks good but want to hear your explanation.